dump/dump_watch.cpp

```diff
@@ -96,6 +105,15 @@ void DumpWatch::interfaceRemoved(sdbusplus::message::message& msg)
         sdbusplus::message::object_path objPath;
         DBusInteracesList interfaces;
         msg.read(objPath, interfaces);
+        if (std::find(interfaces.begin(), interfaces.end(),
+                      "com.ibm.Dump.Entry.Hostboot") == interfaces.end() &&
+            std::find(interfaces.begin(), interfaces.end(),
+                      "com.ibm.Dump.Entry.Hardware") == interfaces.end() &&
+            std::find(interfaces.begin(), interfaces.end(),
+                      "com.ibm.Dump.Entry.SBE") == interfaces.end() &&
+            std::find(interfaces.begin(), interfaces.end(),
+                      "xyz.openbmc_project.Dump.Entry.BMC") == interfaces.end())
+            return;
```

Comment:
if you remove in offload_manager.cpp it will not come here, no need special handling here, also you are not removing BMC in offload_manager but including here.